loc_scraper.gemspec

```diff
@@ -15,8 +15,8 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
-  gem.add_dependency 'mechanize', '~> 2.7'
-  gem.add_dependency 'mime-types', '~> 1.25'
+  gem.add_dependency 'mechanize', '~> 2.8.0'
```

Comment:
Do the dependencies for `mechanize` and `mime-types` need to be more relaxed? I tried setting these to `~> 2.8` and `mime-types` to `~> 3.0` and that caused `2.8.5` and `3.5.2` to be installed respectively which I didn't want.